src/script/ui/component/view/icon.jsx

```diff
@@ -17,6 +17,9 @@
 import { h } from 'preact';
 
 function Icon({ name, ...props }) {
+	// TODO: correct
+	if (name === 'enhanced-stereo')
+		name = 'cip';
```

Comment:
Please add svg file for this Icon and remove this statement

Reply:
Done